generators/python-v2/ast/src/TypeInstantiation.ts

```diff
@@ -200,6 +204,20 @@ export class TypeInstantiation extends AstNode {
                 assertNever(this.internalType);
         }
     }
+
+    private writeStringWithTripleQuotes({ writer, value }: { writer: Writer; value: string }): void {
+        writer.write('"""');
+        const parts = value.split("\n");
+        const head = parts[0] + "\n";
+        const tail = parts.slice(1).join("\n");
+        writer.write(this.escapeString(head));
+        writer.writeNoIndent(this.escapeString(tail));
+        writer.write('"""');
+    }
+
+    private escapeString(value: string): string {
+        return value.replaceAll('"', '\\"');
```

Comment:
Tbh I'm not sure if this is the right escape strategy here.

Reply:
I wouldn't expect to need this during multiline strings